networkit/cpp/graph/Graph.cpp

```diff
@@ -771,6 +771,9 @@ void Graph::removeEdge(node u, node v) {
     if (weighted) {
         erase<edgeweight>(u, vi, outEdgeWeights);
     }
+    if (edgesIndexed) {
+        erase(u, vi, outEdgeIds);
+    }
```

Comment:
I see that it is not necessary to specify the template parameter of `erase`. I think that it would be better to remove it when used, at least within this function, otherwise it is a bit confusing for the reader.